common/models/user.js

```diff
@@ -573,6 +575,19 @@ module.exports = function(User) {
     this.settings.maxTTL = this.settings.maxTTL || DEFAULT_MAX_TTL;
     this.settings.ttl = this.settings.ttl || DEFAULT_TTL;
 
+    // user email case sensitivity default to true
+    if (this.settings.caseSensitiveEmail === undefined) {
```

Comment:
I couldn't change the setting in a boot script if I set it to true in user.js. Maybe I'm just trying to set it in the wrong place? Do you have a line number you could refer me to?